components/link/css/template.js

```diff
@@ -53,5 +53,5 @@ export const Link = ({
   })}"${external ? ' rel="external noopener noreferrer"' : ''}${current ? ' aria-current="page"' : ''}${
     title ? ` title="${title}"` : ''
   }${hreflang ? ` hreflang="${hreflang}"` : ''}${lang ? ` lang="${lang}"` : ''}${
-    !current ? ' rel="alternate"' : ''
+    hreflang && !current ? ' rel="alternate"' : ''
```

Comment:
`hreflang` kun je altijd gebruiken voor linkjes, niet alleen voor linkjes naar een alternatieve versie van dezelfde pagina.

Bijvoorbeeld:

```html
<ul title="Government Design Systems">
  <li><a href="https://design-system.service.gov.uk" hreflang="en-GB">United Kingdom</a></li>
  <li><a href="https://designers.italia.it" hreflang="it">Italy</a></li>
</ul>
```

Lijkt me daarom niet handig om `alternate` af te leiden van `lang` of `hreflang`.